### PR TITLE
[infra] Skip code coverage job for projects that are disabled (#1743, #1547).

### PR DESCRIPTION
**Pull request by Dor1s**
Another minor follow-up.

### CODE REVIEW

File: infra/gcb/build_and_run_coverage.py [hunk 1]
```diff
@@ -32,6 +32,14 @@
 UPLOAD_REPORT_URL_FORMAT = 'gs://%s/{0}/reports/{1}' % COVERAGE_BUCKET_NAME
 
 
+def skip_build(message):
+  # Exit with 0 not to mark code coverage job as failed.
+  # Since the script should print build_id, print '0' as a special value.
+  sys.stderr.write('%s\n' % message)
+  print '0'
```
Comment (inferno-chromium) [inline infra/gcb/build_and_run_coverage.py]: Is this print needed now ?

File: infra/gcb/build_and_run_coverage.py [hunk 2]
```diff
@@ -32,6 +32,14 @@
 UPLOAD_REPORT_URL_FORMAT = 'gs://%s/{0}/reports/{1}' % COVERAGE_BUCKET_NAME
 
 
+def skip_build(message):
+  # Exit with 0 not to mark code coverage job as failed.
```
Comment (inferno-chromium) [inline infra/gcb/build_and_run_coverage.py]: Change # to """

Comment (inferno-chromium) [reply infra/gcb/build_and_run_coverage.py]: Also this comment needs fixing this is just not for coverage job failure.

Comment (Dor1s) [reply infra/gcb/build_and_run_coverage.py]: Done! This is for coverage job only (filename is `build_and_run_coverage.py`) :)

File: infra/gcb/build_and_run_coverage.py (hunk 1 above)
Comment (Dor1s) [reply infra/gcb/build_and_run_coverage.py]: Yes, it's still needed, at least while we're still using Jenkins with `wait_for_build.py` helper script.

**Pull request merged**